app/src/main/java/com/android/shelfLife/model/recipe/RecipeGeneratorOpenAIRepository.kt

```diff
@@ -32,193 +27,186 @@ import kotlinx.serialization.json.putJsonArray
 import kotlinx.serialization.json.putJsonObject
 
 @Singleton
-class RecipeGeneratorOpenAIRepository
-@Inject
-constructor(private val openai: OpenAI, @IoDispatcher private val dispatcher: CoroutineDispatcher) :
+class RecipeGeneratorOpenAIRepository @Inject constructor(private val openai: OpenAI) :
```

Comment:
This way of using hilt is much better in my opinion. So thank you for changing it.